pkg/kubectl/cmd/taint.go

```diff
@@ -115,18 +113,15 @@ func NewCmdTaint(f cmdutil.Factory, out io.Writer) *cobra.Command {
 
 // reorganizeTaints returns the updated set of taints, taking into account old taints that were not updated,
 // old taints that were updated, old taints that were deleted, and new taints.
-func reorganizeTaints(accessor metav1.Object, overwrite bool, taintsToAdd []v1.Taint, taintsToRemove []v1.Taint) ([]v1.Taint, error) {
-	newTaints := append([]v1.Taint{}, taintsToAdd...)
-
-	var oldTaints []v1.Taint
-	var err error
-	annotations := accessor.GetAnnotations()
-	if annotations != nil {
-		if oldTaints, err = v1.GetTaintsFromNodeAnnotations(annotations); err != nil {
-			return nil, err
-		}
+func reorganizeTaints(obj runtime.Object, overwrite bool, taintsToAdd []v1.Taint, taintsToRemove []v1.Taint) ([]v1.Taint, error) {
+	node, ok := obj.(*v1.Node)
+	if !ok {
+		return nil, fmt.Errorf("unexpected type %T", obj)
```

Comment:
can you add at the end of the message "expected Node" (though it will be obvious if someone traces down the source code line)